site/content/docs/multistep-checks/example-checks.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Example checks
+weight: 22
+menu:
+  resources:
+    parent: "Multistep checks"
+    identifier: "multistep-example-checks"
+cli: true
+---
+
+Below are a number of checks showcasing how to use Multistep checks. These are similar to the templates provided in Checkly when creating a new Multistep check and should give you an idea of how to get started building various types of API monitors.
+
+Some of the examples require setting up authentication tokens to function.
+
+### API authentication and CRUD testing
```

Comment:
@drakirnosslin the example itself looks helpful, but I think having these kinda "recursive" examples where you are teaching someone to write a Checkly check against Checkly can confuse less experienced users (where does the check end and the target system begin?), I try not to use those. This might be less of a problem given most of our users are technical

Reply:
So I am not asking for any action here, just food for thought.